src/app/PersonOppslagHandler/LyttPåNyttFnrIReduxOgHentAllPersoninfo.tsx

```diff
@@ -0,0 +1,66 @@
+import { hentKontaktinformasjon } from '../../redux/restReducers/kontaktinformasjon';
+import { erEgenAnsatt } from '../../redux/restReducers/egenansatt';
+import { hentVergemal } from '../../redux/restReducers/vergemal';
+import { hentFeatureToggles } from '../../redux/restReducers/featureToggles';
+import { resetNavKontorReducer } from '../../redux/restReducers/navkontor';
+import { resetUtbetalingerReducer } from '../../redux/restReducers/utbetalinger';
+import { resetSykepengerReducer } from '../../redux/restReducers/ytelser/sykepenger';
+import { resetPleiepengerReducer } from '../../redux/restReducers/ytelser/pleiepenger';
+import { resetForeldrepengerReducer } from '../../redux/restReducers/ytelser/foreldrepenger';
+import { resetUtførteUtbetalingerReducer } from '../../redux/restReducers/ytelser/utførteUtbetalinger';
+import { resetKontrollSpørsmål } from '../../redux/kontrollSporsmal/actions';
+import { hentPerson } from '../../redux/restReducers/personinformasjon';
+import { useEffect } from 'react';
+import { AppState } from '../../redux/reducers';
+import { connect } from 'react-redux';
+import { AsyncDispatch } from '../../redux/ThunkTypes';
+
+interface StateProps {
+    fnrIKontekst?: string;
+}
+
+interface DispatchProps {
+    oppslagNyBruker: (fnr: string) => void;
+}
+
+type Props = StateProps & DispatchProps;
+
+function LyttPåNyttFnrIReduxOgHentAllPersoninfo(props: Props) {
+    useEffect(() => {
```

Comment:
Denne ser så enkel og elegant ut, blir nesten stressa for at vi ikke har tenkt på corner cases?

Reply:
mm, har prøvd å teste den på mange forskjellige måter, ikke skjedd noe rart enda..